src/System.Net.Http.WinHttpHandler/tests/FunctionalTests/WinHttpHandlerTest.cs

```diff
@@ -107,6 +110,24 @@ public void SendAsync_SlowServerRespondsAfterDefaultReceiveTimeout_ThrowsHttpReq
             }
         }
 
+        [ConditionalTheory(nameof(IsWindows10Version1607)), MemberData(nameof(Http2Servers))]
```

Comment:
The current test in System.Net.Http.Functional.Tests is not very specific.  It tests for either HTTP/2 or HTTP/1.1.  I wanted a very specific test that always tests for getting HTTP/2 assuming it is running on a supported Windows 10 OS.